package.json

```diff
@@ -229,6 +230,13 @@
       }
     ],
     "menus": {
+      "commandPalette": [
+        {
+          "command": "extension.NAMESPACE(node-debug).prettyPrint",
+          "title": "%pretty.print.script%",
+          "when": "inDebugMode && debugType == NAMESPACE(chrome) && editorLangId == javascript"
```

Comment:
Why just `debugType == NAMESPACE(chrome)`?

Reply:
The child sessions use chrome as their 'fake' debug type in the attachment process. So that check will return true any time there's at least one debug target :)

https://github.com/microsoft/vscode-js-debug/blob/2f8893afbcd9b1f67f42cbaa20d7b96d7969b337/src/sessionManager.ts#L187-L192